expand.cpp

```diff
@@ -493,7 +496,7 @@ namespace Sass {
       // { target_vec.push_back((*compound_sel)[i]->perform(&to_string)); }
       for (size_t i = 0, L = extender->length(); i < L; ++i) {
         // let's test this out
-        // cerr << "REGISTERING EXTENSION REQUEST: " << (*extender)[i]->perform(&to_string) << " <- " << compound_sel->perform(&to_string) << endl;
+         cerr << "REGISTERING EXTENSION REQUEST: " << (*extender)[i]->perform(&to_string) << " <- " << compound_sel->perform(&to_string) << endl;
```

Comment:
This one is not my own, it's from the code base, but is commented out. I re-comment it out